modules/installation-disk-partitioning-upi-templates.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-user-infra.adoc
+// * installing/installing_aws/installing-restricted-networks-aws.adoc
+// * installing/installing_azure/installing-azure-user-infra.adoc
+// * installing/installing_gcp/installing-gcp-user-infra.adoc
+// * installing/installing_gcp/installing-restricted-networks-gcp.adoc
+
+// Similar content to what is in this module is also present in modules/installation-disk-partitioning.adoc. <-- This module is in use with the following vSphere assemblies:
+//    * installing-vsphere.adoc
+//    * installing-vsphere-network-customizations.adoc
+//    * installing-restricted-networks-vsphere.adoc
+
+// Similar content to what is in this module is also present in modules/installation-user-infra-machines-advanced.adoc. <-- This module is in use with the following bare metal assemblies:
+//    * installing-bare-metal-network-customizations.adoc
+//    * installing-bare-metal.adoc
+//    * installing-restricted-networks-bare-metal.adoc
+
+[id="installation-disk-partitioning-upi-templates_{context}"]
+= Creating a separate `/var` partition
+In general, disk partitioning for {product-title} should be left to the
```

Comment:
> ! should v
Always prefer a more direct verb or way of saying something. Omit “should,” or use “must” or “might,” as appropriate.

Reply:
Revised to omit "should" here.